src/DacpacTool/VersionProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Reflection;
+using NuGet.Versioning;
+
+namespace MSBuild.Sdk.SqlProj.DacpacTool
+{
+    public class VersionProvider : IVersionProvider
+    {
+        public NuGetVersion CurrentPackageVersion()
+        {
+            return new NuGetVersion(
+                Assembly
+                .GetEntryAssembly()!
```

Comment:
Wouldn't this be the same as `typeof(VersionProvider).Assembly`?

Reply:
It would